repos/spack_repo/builtin/packages/sphexa/package.py

```diff
@@ -45,16 +58,21 @@ class Sphexa(CMakePackage, CudaPackage, ROCmPackage):
 
         depends_on("mpich +rocm", when="+rocm ^[virtuals=mpi] mpich")
 
-    conflicts("%gcc@:10")
+    conflicts("%gcc@:11", when="@0.95:")
+    conflicts("%gcc@:10", when="@:0.93.1")
```

Comment:
double checking, this means that:
- new versions starting from 0.95 (included) do not build with `gcc@11` or older, i.e. require at least `gcc@12`
- old versions up to 0.93.1 do not build with `gcc@10` or older, i.e. require at least `gcc@11`